guides/common/modules/proc_displaying-rhel-lifecycle-status.adoc

```diff
@@ -0,0 +1,13 @@
+[id="Displaying_RHEL_Lifecycle_Status_{context}"]
+= Displaying RHEL Lifecycle Status
+
+You can display the status of the end of support (EOS) for your {RHEL} hosts in the table on the Hosts index page.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Click *Manage columns*.
+. Select the *Content* column to expand it.
+. Select *RHEL Lifecycle status*.
+. Click *Save* to generate a new column that displays the {EL} lifecycle status.
+
+For any hosts that are not running {EL}, *Unknown* displays in the *RHEL Lifecycle status* and *Last report* columns.
```

Comment:
I don't think that we need to have this sentence twice. I'm leaning towards removing this one.